client/src/main/java/tds/assessment/ItemProperty.java

```diff
@@ -10,18 +10,10 @@
     private String description;
 
     public ItemProperty(String name, String value, String description, String itemId) {
```

Comment:
These other ones didn't make sense because if item id isn't populated other places in the system throw NPE's.